templates/quiz/start-button.php

```diff
@@ -53,25 +54,51 @@
 						 * @param LLMS_Quiz   $quiz        The current quiz instance.
 						 * @param LLMS_Lesson $lesson      The parent lesson instance.
 						 */
-						echo apply_filters( 'lifterlms_begin_quiz_button_text', __( 'Start Quiz', 'lifterlms' ), $quiz, $lesson );
+						echo apply_filters( 'lifterlms_begin_quiz_button_text', esc_html( $start_button_string ), $quiz, $lesson );
 					?>
 				</button>
+			</form>
+		<?php else : ?>
+			<p><?php esc_html_e( 'You are not able to take this quiz', 'lifterlms' ); ?></p>
+		<?php endif; ?>
+
+		<?php if ( $quiz->is_open() && $quiz->can_be_resumed_by_student() ) : ?>
+			<form method="POST" action="" name="llms_resume_quiz" enctype="multipart/form-data">
 
-			<?php else : ?>
-				<p><?php _e( 'You are not able to take this quiz', 'lifterlms' ); ?></p>
-			<?php endif; ?>
+				<input id="llms-attempt-key" name="llms_attempt_key" type="hidden" value="<?php echo $quiz->get_student_last_attempt_key(); ?>"/>
 
-			<?php if ( $lesson->get_next_lesson() && llms_is_complete( get_current_user_id(), $lesson->get( 'id' ), 'lesson' ) ) : ?>
-				<a href="<?php echo get_permalink( $lesson->get_next_lesson() ); ?>" class="button llms-button-secondary llms-next-lesson"><?php _e( 'Next Lesson', 'lifterlms' ); ?></a>
-			<?php endif; ?>
-		</form>
+				<input type="hidden" name="action" value="llms_resume_quiz" />
+
+				<?php wp_nonce_field( 'llms_resume_quiz' ); ?>
+
+				<button class="llms-resume-quiz-button llms-button-action button" id="llms_resume_quiz" name="llms_resume_quiz" type="submit">
+					<?php
+						/**
+						 * Filters the quiz resume button text.
+						 *
+						 * @since [version]
+						 *
+						 * @param string      $button_text The resume quiz button text.
+						 * @param LLMS_Quiz   $quiz        The current quiz instance.
+						 * @param LLMS_Lesson $lesson      The parent lesson instance.
+						 */
+						echo esc_html( apply_filters( 'lifterlms_resume_quiz_button_text', __( 'Resume Quiz', 'lifterlms' ), $quiz, $lesson ) );
```

Comment:
https://github.com/gocodebox/lifterlms/blob/trunk/docs/coding-standards.md#hooks-actions--filters

It should be `llms_resume_quiz_button_text`
plus, please, escape the html.
I know that the other filter is `lifterlms_begin_quiz_button_text`, but still new added filters have to use the correct prefix. We can think of adding a new filter hook `llms_begin_quiz_button_text` and deprecate the old one in the future.

Reply:
got it. Already noted it but the change was made before I got to know about this 😅

Reply:
:D